examples/E2EBert/news_classifier.py

```diff
@@ -416,5 +416,4 @@ def configure_optimizers(self):
     trainer.fit(model, dm)
     trainer.test()
     if trainer.global_rank == 0:
-        with mlflow.start_run() as run:
-            mlflow.pytorch.save_state_dict(trainer.get_model().state_dict(), ".")
+        mlflow.pytorch.save_state_dict(trainer.get_model().state_dict(), ".")
```

Comment:
@shrinath-suresh Has this been tested for Distributed training run with multiple workers on same machine? Previously start_run was only called for the master "global_rank 0" worker

Reply:
@chauhang Yes Geeta. Even now, the save model will be called only for rank 0. 

Only difference being

We don't have to start a new run. Here is the work flow 

1. User runs the model using `mlflow run .` - A new run will be created
2. `mlflow.pytorch.autolog` - uses the existing run and **doesn't end the run**
3. We can simply save the model using `mlflow.pytorch.save_state_dict` . No need for creating a new run.

Reply:
@KasirajanA Can you please test these examples once in GPU and attach the screenshot

Reply:
@shrinath-suresh, @chauhang Both the MNIST and E2Ebert examples ran successfully in gpu instance. Please find the attached screenshots

MNIST
---------
![MNIST_train](https://user-images.githubusercontent.com/83957636/118983241-d8996b00-b999-11eb-8e4c-b5067e38c6c2.png)

![MNIST_prediction](https://user-images.githubusercontent.com/83957636/118983236-d6cfa780-b999-11eb-81d6-46bb33661874.png)


E2EBert
------------
![E2Ebert_train](https://user-images.githubusercontent.com/83957636/118983287-e51dc380-b999-11eb-85db-c6911159df93.png)

![E2Ebert_predict](https://user-images.githubusercontent.com/83957636/118983306-e8b14a80-b999-11eb-9d63-92e89eaf56cc.png)